### PR TITLE
Enable ITS encryption by default on 54L + other TF-M updates

### PR DESCRIPTION
**Pull request by mswarowsky**

### CODE REVIEW

File: samples/crypto/persistent_key_usage/README.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ Crypto: Persistent key storage
 
 The persistent key sample shows how to generate a persistent key using the Platform Security Architecture (PSA) APIs.
 Persistent keys are stored in the Internal Trusted Storage (ITS) of the device and retain their value between resets.
+The ITS backend is either provided by TF-M or the :ref:`trusted_storage_readme` library for non TF-M builds.
```
Comment (mia-ko) [inline samples/crypto/persistent_key_usage/README.rst:12]:
```suggestion
The ITS backend is either provided by TF-M, or the :ref:`trusted_storage_readme` library when building applications without TF-M.
```

File: samples/crypto/persistent_key_usage/README.rst [hunk 2]
```diff
@@ -41,6 +42,9 @@ The sample performs the following operations:
    The key ID of a read-only key is writable again after a full erase of the device memory.
    Use the ``west -v flash --erase`` command for the full erase.
 
+.. note::
+   For non-TF-M builds, and on nRF54L15 devices, the :ref:`hardware unique key (HUK) <lib_hw_unique_key>` is used to encrypt the key before storing it.`
```
Comment (mia-ko) [inline samples/crypto/persistent_key_usage/README.rst:46]:
```suggestion
   In environments without TF-M and on nRF54L15 devices, the :ref:`hardware unique key (HUK) <lib_hw_unique_key>` is used to encrypt the key before storing it.`
```

File: samples/crypto/persistent_key_usage/README.rst [hunk 3]
```diff
@@ -56,3 +60,14 @@ After programming the sample to your development kit, complete the following ste
...
 #. Observe the logs from the application using a terminal emulator.
+
+Dependencies
+************
+
+* PSA APIs:
+
+   * :file:`psa/crypto.h`
+
+* Non TF-M builds use the :ref:`trusted_storage_readme` library
```
Comment (mia-ko) [inline samples/crypto/persistent_key_usage/README.rst:71]:
```suggestion
* Builds without TF-M use the :ref:`trusted_storage_readme` library
```

**Review by mia-ko — CHANGES_REQUESTED** [review]
Rewriting when non TF-M is used...
because it should be hyphenated 'non-TF-M', but that just looks a bit odd :)
Let me know if what I used instead doesn't feel right either.

**Review by vili-nordic — APPROVED** [review]

File: samples/crypto/persistent_key_usage/README.rst (hunk 2 above)
Comment (mswarowsky) [reply samples/crypto/persistent_key_usage/README.rst]: Discussed offline an changed

**Review by mia-ko — APPROVED** [review]

**Review by Vge0rge — APPROVED** [review]

**Pull request merged**